Makefile

```diff
@@ -99,6 +100,26 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	stg repair
 	stg import -s ../patch/series
 
+	mkdir $(NON_UP_LOC)
```

Comment:
Can you add a line to remove this directory before creating it, in case it exists already?

Reply:
This sits under BUIlD_DIR and we delete that in the beginning, so this isn’t a problem